javaslang/src-gen/main/java/javaslang/Tuple.java

```diff
@@ -95,6 +96,15 @@ static Tuple0 empty() {
     }
 
     /**
+    * Creates a tuple of two elements from java.util.Map.Entry.
+    * @param entry A java.util.Map.Entry
+    * @return a tuple of two elements.
+    */
+     static <T1, T2> Tuple2<T1, T2> fromEntry(Map.Entry<T1, T2> entry) {
```

Comment:
_Only immutable types are covariant in generic arguments_ - Nice 👍 
@danieldietrich will make the changes and thanks for the example 😄